source/partials/release_notes/_release-19-10-0.md.erb

```diff
@@ -0,0 +1,81 @@
+<h4>Improved Agents page</h4>
+
+Agents page has been improved in this release. Now it will have 2 tabs - Static and Elastic - for easy viewing and searching of static and elastic agents.
+
+<figure class='shrink-figure-100'>
+    <%= image_tag 'releases/19.10.0/new-agents-page-static-agents.png', alt: 'New Agents page - Static agents' %>
+</figure>
+
+<figure class='shrink-figure-100'>
+    <%= image_tag 'releases/19.10.0/new-agents-page-elastic-agents.png', alt: 'New Agents page - Elastic agents' %>
+</figure>
+
+<h4>Delete DB cruft during upgrade</h4>
+
+* <%= link_to_commit '7d5d01f55320d7d70ec80d4af646120635842f61', "Delete cruft from the DB for jobs that are completed" %>
+
+There are quite some unwanted data present in the DB tables `environmentVariables`, `artifactPlans`, `resources`, `jobAgentMetadata` for completed jobs.
```

Comment:
Add a note about taking a backup before upgrading. It is standard procedure but worth stressing on here.